qiita_db/metadata_template/util.py

```diff
@@ -193,33 +191,7 @@ def load_template_to_dataframe(fn, strip_whitespace=True, index='sample_name'):
     character will be ignored and columns that are empty will be removed. Empty
     sample names will be removed from the DataFrame.
 
-    The following table describes the data type per column that will be
-    enforced in `fn`. Column names are case-insensitive but will be lowercased
-    on addition to the database.
-
-    +-----------------------+--------------+
-    |      Column Name      |  Python Type |
-    +=======================+==============+
-    |           sample_name |          str |
-    +-----------------------+--------------+
-    |             #SampleID |          str |
-    +-----------------------+--------------+
-    |     physical_location |          str |
-    +-----------------------+--------------+
-    | has_physical_specimen |         bool |
-    +-----------------------+--------------+
-    |    has_extracted_data |         bool |
-    +-----------------------+--------------+
-    |           sample_type |          str |
-    +-----------------------+--------------+
-    |       host_subject_id |          str |
-    +-----------------------+--------------+
-    |           description |          str |
-    +-----------------------+--------------+
-    |              latitude |        float |
-    +-----------------------+--------------+
-    |             longitude |        float |
-    +-----------------------+--------------+
+    Everything in the DataFrame will be read and managed as string
```

Comment:
Unless this is no longer the case, can you re-add: `Column names are case-insensitive but will be lowercased on addition to the database.`?